src/libraries/Microsoft.Extensions.HostFactoryResolver/pkg/Microsoft.Extensions.HostFactoryResolver.Sources.pkgproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <PropertyGroup>
+    <IsShipping>false</IsShipping>
+  </PropertyGroup>
+  <ItemGroup>
+    <SourcePackageFiles Include="..\src\*.*" PackagePath="contentFiles/cs/netstandard1.0/%(FileName)%(Extension)"  />
```

Comment:
This matches what dotnet/extensions was doing.  I tested the resulting package and it behaves as before.